hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -191,7 +191,9 @@ def cached_strategy(*args, **kwargs):
     return cached_strategy
 
 
-def base_defines_strategy(force_reusable: bool) -> Callable[[T], T]:
+def base_defines_strategy(
+    force_reusable: bool, *, force_lazy: bool = True
```

Comment:
`force_lazy` is both unexplained in the docstring and probably a bad name for it. In particular the use of "force" means kinda the opposite of what it means in `force_reusable` - in the latter it means "force this thing to be reusable regardless of whether it looks like it should be" while in the former it means "force the evaluation of this lazy value, i.e. force this strategy to be non-lazy"